tests/Benchmarks/Benchmarks.csproj

```diff
@@ -31,6 +31,7 @@
     <PackageReference Include="BenchmarkDotNet.Diagnostics.Windows" Version="$(BenchmarkDotNetVersion)" />
     <PackageReference Include="Microsoft.Diagnostics.Tracing.TraceEvent" Version="2.0.30" />
     <PackageReference Include="System.Text.Encodings.Web" Version="$(CoreFxStableVersion)" />
+    <PackageReference Include="xunit" Version="$(XUnitVersion)" IsImplicitlyDefined="true" PrivateAssets="all" Publish="true"/>
```

Comment:
Make sure we still need this, as I believe that because you set IsTestProject on all projects under tests, I would imagine this reference should be fetched automatically.

Reply:
Testing without this ref. Don't remember why this was needed anymore.